drivers/project_scorecard/app/models/project_scorecard/project_performance.rb

```diff
@@ -54,6 +54,8 @@ def exit_to_ph_score
           score(exit_to_ph_percentage, 98..100, 90..97)
         elsif project.rrh?
           score(exit_to_ph_percentage, 95..100, 90..94)
+        else
```

Comment:
What is the case this should happen? Scorecards are limited to PSH/RRH projects, so if we encounter this case, I wonder if something else is causing it to fall through?

Reply:
Yes, the use case is currently RRH/PSH, this is just to give a sane default when they begin using it for ES/TH